### ISSUE TITLE
Name property is not defined for multiple controls in the header section.

### ISSUE THREAD

**Issue by JamesFerebee** [microsoft/SqlNexus#343]
Same issue replicates for similar controls throughout the application.

Impact on a Person with Disability (PWD):
Screen Readers users are getting impacted and will not understand the context of given control as name Property is not defined for multiple elements.

Test Environment:​

OS: Windows 11 [Version 22H2 (OS Build 22621.2283)].

Desktop application: SQL Nexus.

Tool: Narrator, JAWS (2022.2110.6).

Prerequisite:
1. Install SQLNexus application in the desktop.
2. Turn on Accessibility Insights for windows.
3. Enable Narrator/JAWS.

Repro Steps:
Step 1: Open the SQL Nexus app on the desktop.
Step 2: Connect to server Dialog gets displayed. Connect to the server by invoking connect button.
Step 3: Navigate to SQL Perf Main and invoke it.
Step 4: Navigate to Best practices & Analysis summary and invoke it.
Step 5: Run accessibility insights for Windows tool and Verify whether the Name property is not defined for multiple controls in the header section or not
 
Actual Result:
Name property is not defined for multiple controls in the header section.
1. 100 % combo box
2. Find edit field.
3. Current DB combo box
Screen reader behavior: 
1. Narrator is announcing as '100% combo box collapsed'.
2. Narrator is announcing as 'Edit'.
3. Narrator is announcing as 'SQLNexus combo edit collapsed SQLNexus selected'.

Expected Result:
Name property should be defined for above mentioned controls in the header section.
Ex Screen reader behavior: 
1. Narrator should be announcing as 'zoom 100% combo box collapsed'.
2. Narrator should be announcing as 'Find Edit field'.
3. Narrator should be announcing as 'Current DB SQLNexus combo edit collapsed SQLNexus selected'.

Suggestive Fixes:
Provide "aria-label" attribute with required name to the elements.

**Comment by JamesFerebee** [comment]
1711725

**Comment by PiJoCoder** [comment]
Fixed by PR https://github.com/microsoft/SqlNexus/pull/365
